glass/cosmology.py

```diff
@@ -0,0 +1,25 @@
+"""Module for cosmology.api utilities."""
+
+from typing import Any, Protocol
+
+from numpy.typing import NDArray
+
+import cosmology.api
+
+Array = NDArray[Any]
+
+
+class Cosmology(
+    cosmology.api.HasComovingDistance[Array, Array],  # type: ignore[misc]
```

Comment:
`mypy` doesn't like these. Fails with `Class cannot subclass value of type "Any"`. Have taken from the examples in the `cosmology.api` documentation https://cosmology.readthedocs.io/projects/api/latest/quickstart.html#combining-protocols.

Reply:
Maybe @ntessore and @nstarman can look at this 👀